hack/patch-crd/go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/elastic/cloud-on-k8s/hack/strip-podTemplate-description
```

Comment:
should we rename the module to match the path? `hack/patch-crd`?

Reply:
Yes. Previous first name is now revealed.